cernopendata/modules/fixtures/data/docs/atlas-open-data-for-research-release-2024/atlas-open-data-for-research-release-2024.md

```diff
@@ -4,21 +4,21 @@ The ATLAS Experiment at CERN has made two years’ worth of scientific data avai
 
-Today’s release builds upon previous open data releases for educational use (in 2016 and 2020). “All of our open data releases are now available through the [ATLAS open data website](https://opendata.atlas.cern/),” says Dilia Portillo, ATLAS Outreach and Education Coordinator. “The website includes multi-level documentation, video tutorials and online tools aimed at the full-spectrum of users, from high school students to senior particle physics researchers. In addition, the software used to create the education-use open data has been released. This provides a seamless transition from the research open data to all the tutorials for outreach and education, including newly updated Higgs-boson discovery documentation. With a bit of time and dedication, you can go from being a relative novice to carrying out your own analysis.”
+Today’s release builds upon previous open data releases for educational use (in 2016 and 2020). “All of our open data releases are now available through the [ATLAS open data website](https://opendata.atlas.cern/),” says Dilia Portillo, ATLAS Outreach and Education co-Coordinator. “The website includes multi-level documentation, video tutorials and online tools aimed at the full-spectrum of users, from high school students to senior particle physics researchers. In addition, the software used to create the education-use open data has been released. This provides a seamless transition from the research open data to all the tutorials for outreach and education, including newly updated Higgs-boson discovery documentation. With a bit of time and dedication, you can go from being a relative novice to carrying out your own analysis.”
 
 The ATLAS open data website also serves as a hub for the community, which includes teachers, students, enthusiasts and, now, scientists. Anyone diving into the open data can also directly engage with ATLAS physicists, who are available to respond to user feedback and take suggestions.
 
 This release marks the start of more to come, with ATLAS’ first release of lead-lead-nuclei collision data data up next. The ATLAS Collaboration, along with the other main LHC experiment collaborations, has [committed](https://home.cern/news/press-release/knowledge-sharing/cern-announces-new-open-data-policy-support-open-science) to making all of its data publicly accessible after a certain time. Openness is deeply ingrained in the culture of high-energy physics, enabling greater accessibility, reproducibility and better science.
 
 Begin your journey with [ATLAS open data](https://opendata.atlas.cern/docs/userpath/quickstart) by following the tutorial below.
 
-[![ATLAS Open Data Tutorial](https://img.youtube.com/vi/p36hfgy2b9E/0.jpg)](https://www.youtube.com/watch?v=p36hfgy2b9E)
+[![ATLAS Open Data Tutorial](/static/docs/atlas-open-data-for-research-release-2024/0.jpg)](https://www.youtube.com/watch?v=p36hfgy2b9E)
```

Comment:
Please update the link to:
https://www.youtube.com/watch?v=r83e1FJOxqg

Reply:
Done.